server/jvm/{{appName}}-dictionary-cache/build.gradle.kts

```diff
@@ -1,13 +1,2 @@
 
-// Add your genesis config dependencies here
-dependencies {
-    implementation(project(":{{appName}}-dictionary-cache:{{appName}}-generated-dao"))
-    implementation(project(":{{appName}}-dictionary-cache:{{appName}}-generated-fields"))
-    implementation(project(":{{appName}}-dictionary-cache:{{appName}}-generated-hft"))
-    implementation(project(":{{appName}}-dictionary-cache:{{appName}}-generated-sysdef"))
-    implementation(project(":{{appName}}-dictionary-cache:{{appName}}-generated-view"))
-
-    implementation("global.genesis:auth-config:${properties["authVersion"]}")
```

Comment:
Looks like by default we add the dependency on auth. We should probably add it as a default dependency in the genesis settings in settings.gradle.kts